workflows/social/LightTest.bonsai

```diff
@@ -0,0 +1,92 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<WorkflowBuilder Version="2.8.1"
+                 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                 xmlns:rx="clr-namespace:Bonsai.Reactive;assembly=Bonsai.Core"
+                 xmlns:harp="clr-namespace:Bonsai.Harp;assembly=Bonsai.Harp"
+                 xmlns:aeon-env="clr-namespace:Aeon.Environment;assembly=Aeon.Environment"
+                 xmlns="https://bonsai-rx.org/2018/workflow">
+  <Workflow>
+    <Nodes>
+      <Expression xsi:type="GroupWorkflow">
+        <Name>FakeSynchronizer</Name>
+        <Workflow>
```

Comment:
@aspaNeuro  Why not simply use the synchronizer on AEON3? I guess my thinking here is that we would calibrate all the lights in the room at the same time from AEON3 where the server is, there is no need to do this remotely, but maybe I am missing something?